README.md

```diff
@@ -39,13 +38,13 @@ For Scala 2.11.x and 2.12.x:
 [comment]: # (Start Replace)
 
 ```scala
-libraryDependencies += "com.47deg" %% "iota-core" % "0.0.2"
+libraryDependencies += "com.47deg" %% "iota-core" % "0.1.0"
```

Comment:
You don't need to do this, sbt-org-policies will replace the version automatically.